### ISSUE TITLE
`nix` crate failed to build on `loongarch64` architecture

### ISSUE THREAD

**Issue by eatradish** [rust-disk-partition-management/gptman#134]
`nix` crate must be at least version 0.27.0 to compile in `loongarch64` , But I noticed that `Cargo.toml` has a version range for nix:

```toml
[target.'cfg(target_os = "linux")'.dependencies]
nix = { version = ">= 0.24, < 0.27", default-features = false, features = ["ioctl"], optional = true }
```

I think upgrading to `nix v0.27.0` shouldn't be a problem.

**Comment by cecton** [comment]
Oh I think you could have done the PR directly, there is not even a lock file anymore

**Comment by cecton** [comment]
Okay I just released 1.1.2 with the wider range and removed the lock file
